composer.json

```diff
@@ -26,7 +26,8 @@
         "beberlei/assert": "^2.0",
         "prooph/common" : "^3.3",
         "react/promise" : "^2.2",
-        "container-interop/container-interop" : "^1.1"
+        "container-interop/container-interop" : "^1.1",
+        "sandrokeil/interop-config": "^0.3"
```

Comment:
can you move container-interop and config-interop to require-dev and suggest, please? the enduser is not forced to use these factories.